mirrord/cli/src/container.rs

```diff
@@ -435,6 +435,19 @@ pub(crate) async fn container_command(
         .status()
         .await;
 
+    // Keep the files, since process is going to exit before process is actually ran and uses it
```

Comment:
```suggestion
    // Keep the files, since the process is going to exit before the new process is actually run and uses them
```
Maybe also be more specific and write which process we're talking about instead of just "the process", like "the internal proxy's process" if that's the process we're talking about.